src/lib/loadErrorPageContent.ts

```diff
@@ -0,0 +1,109 @@
+import gql from "graphql-tag";
+import { print as printQuery } from "graphql";
+
+import { CONTENTFUL_SPACE_ID, CONTENTFUL_DELIVERY_API_TOKEN } from "$env/static/private";
+import getContentfulClient from "$lib/services/contentful";
+import { getBlurhash } from "$lib/services/blurhashes";
+
+import assetProps from "$lib/fragments/assetProps";
+import entryProps from "$lib/fragments/entryProps";
+
+import type { ErrorPageCollectionQuery } from "./$queries.generated";
+import type { ExtractQueryType } from "$lib/util/types";
+
+// TODO: This runs once when the server starts; we should instead run it once per user session.
+let ERROR_CONTENT_LOADED = false;
+const errorPageMap: ErrorPageMap = new Map();
```

Comment:
If y'all have any ideas how we can do this easily, I'm all ears. I tried a few different things but kept running into brick walls. Without individual user session management (which I think is probably out-of-scope here), I'm not sure if there's a way to have this run the way I'd like.